src/firebase/auth/index.ts

```diff
@@ -23,15 +23,15 @@ export const auth = getAuth(app)
  * @param {string} email - new user's e-mail;
  * @param {string} password - new user's password;
  */
-export const createNewUser = async ({ firstName, lastName, role, email, password }: INewUserProps): Promise<void> => {
+export const createNewUser = async ({ firstName, lastName, email, password }: INewUserProps): Promise<void> => {
 	try {
-		const newUser = await createUserWithEmailAndPassword(auth, email, password)
+		const newUser = await createUserWithEmailAndPassword(auth, email as string, password as string)
 
 		await updateProfile(newUser.user, {
 			displayName: `${firstName} ${lastName}`,
 		})
 
-		await setNewUserRoleData({ role: role as ROLES, uid: newUser.user.uid })
+		// await setNewUserRoleData({ role: role as ROLES, uid: newUser.user.uid })
```

Comment:
remove the comments before asking for the review!!! Every fucking time I see this, every time I mention it and then every time you ignore to do thius simple step before asking for the review. 

PLEASE!! I'M BEGGING YOU!!!! REMOVE THE COMMENTS AND CONSOLES BEFORE THE REVIEW!!!!

Reply:
Correct me if I'm wrong, but It seems like this line should remain uncommented, because it looks like the second stage of user creation. first you create it with the email and password, then create and save role for the user, and only then we can call it created user.

Reply:
No. this row shoudn`t exist.
Because setNewUserRoleData function is going to fb/users and create an object {role: role, uid: uid}

But on this time user (includes role and uid) alredy created in 'users' by addNewUser function.

34-th row deleted